build/RELEASING.md

```diff
@@ -44,49 +39,57 @@ Only for Maintainers.
 ```
 
    This generates combined changelog to be used in Github release.
-   Once contents of combined changelog is saved to Github release,
+   Once contents of combined changelog is saved somewhere,
    delete the file.
 
 3.Run the following PowerShell script from the root of the repo.
    This updates all the changelog to have release date for the
    current version being released.
    Replace the version with actual version.
-   The actual version would be the tag name from step1 appended with
-   ".1"
+   The actual version would be the tag name from step1.
 
 ```powershell
      $changelogs = Get-ChildItem -Path . -Recurse -Filter changelog.md
     foreach ($changelog in $changelogs)
     {
      (Get-Content -Path $changelog.FullName) -replace "Unreleased", "Unreleased
 
-## 0.7.0-beta.1
+## 1.0.0-rc2
 
 Released $(Get-Date -UFormat '%Y-%b-%d')" | Set-Content -Path $changelog.FullName
     }
 ```
 
 4.Submit PR with the above changes, and get it merged.
 
-5.Open [Pack and publish to MyGet
+5.Tag Git with version to be released e.g.:
+
+   ```sh
+   git tag -a 1.0.0-rc2 -m "1.0.0-RC2"
+   git push origin 1.0.0-rc2
+   ```
+
+We use [MinVer](https://github.com/adamralph/minver) to do versioning,
+which produces version numbers based on git tags.
+
+6.Open [Pack and publish to MyGet
    workflow](https://github.com/open-telemetry/opentelemetry-dotnet/actions?query=workflow%3A%22Pack+and+publish+to+Myget%22)
    and manually trigger a build. At the end of this, MyGet will have the
-   packages. The package name will be the tag name used in step1 appended with
-   ".1".
+   packages. The package name will be the tag name used in step 5.
 
-6.Validate using MyGet packages. Basic sanity checks :)
+7.Validate using MyGet packages. Basic sanity checks :)
```

Comment:
Do we really need these numbers? They will keep changing.

Reply:
For now yes for easy referrals back and forth. I don't plan to do this manually for long :)
The exact steps, with step numbers are written down to make sure its easy for anyone to write automation for this.